packages/contracts-bedrock/test/dispute/zk/OPSuccinctFaultDisputeGame.t.sol

```diff
@@ -0,0 +1,787 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing
+import "forge-std/Test.sol";
+
+// Libraries
+import { Claim, Duration, GameStatus, GameType, Hash, Timestamp, Proposal } from "src/dispute/lib/Types.sol";
+import {
+    BadAuth,
+    IncorrectBondAmount,
+    UnexpectedRootClaim,
+    NoCreditToClaim,
+    GameNotFinalized,
+    ParentGameNotResolved,
+    InvalidParentGame,
+    ClaimAlreadyChallenged,
+    GameOver,
+    GameNotOver,
+    IncorrectDisputeGameFactory
+} from "src/dispute/lib/Errors.sol";
+import { OP_SUCCINCT_FAULT_DISPUTE_GAME_TYPE } from "src/dispute/lib/Types.sol";
```

Comment:
nit: We can follow up in a later PR. The `OP_SUCCINCT_FAULT_DISPUTE_GAME_TYPE` should be renamed to `OPTIMISTIC_ZK` to match the challenger's.